pkg/controller/servicing/lister.go

```diff
@@ -194,14 +237,24 @@ func (d *NodeLister) Replace() []*core_v1.Node {
 
 	klusterVersion, err := version.ParseSemantic(d.Kluster.Status.ApiserverVersion)
 	if err != nil {
-		d.Logger.Log(
-			"msg", "Couldn't parse Kluster version. Skipping node upgrade.",
-			"err", err,
-		)
-		return found
+		klusterVersion = nil
```

Comment:
I'm confused about this change. When would this make a difference? If the `klusterVersion` is nil it will skip all nodes. Or am I missing something?

Reply:
I doesn't make a real difference, currently nodes get replaced with flatcar even if `klusterVersion` can't be determined.